### PR TITLE
add pyladies page

### PR DESCRIPTION
**Pull request by Leon12-aaryson**
- created the pyladies page

### CODE REVIEW

**Review by HassanBahati — CHANGES_REQUESTED** [review]
@Leon12-aaryson we do have some errors...please fix them. below is a reference of where they are rooted
image

**Comment by HassanBahati** [comment]
> @Leon12-aaryson we do have some errors...please fix them. below is a reference of where they are rooted

Basically all quotation marks (`'` or `"`) needed to be replaced with `&#39;`

**Comment by Leon12-aaryson** [comment]
Ok then, let me go ahead and do that

File: app/2024/pyladies/page.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            <p className="mt-3">Join us in celebrating the PyLadies Anniversary Event at PyCon Uganda!
+              This special occasion marks a significant milestone for PyLadies, an
+              international mentorship group dedicated to empowering women to become
+              active participants and leaders in the Python open-source community. Our mission
+              is to promote, educate, and advance a diverse Python community through
+              outreach, education, conferences, events, and social gatherings.</p>
+          </div>
+          <div className="lg:w-1/2 p-4 w-full">
+            <h1 className="text-3xl">Join Our Anniversary Workshop!</h1>
+            <p className="mt-3">Don&pos;t miss this chance to start your Python journey and join a supportive
```
Comment (HassanBahati) [inline app/2024/pyladies/page.jsx:50]:
```suggestion
            <p className="mt-3">Don&apos;t miss this chance to start your Python journey and join a supportive
```
image

**Review by HassanBahati — CHANGES_REQUESTED** [review]

**Review by HassanBahati — APPROVED** [review]
LGTM! Thank you @Leon12-aaryson 👏

**Comment by Leon12-aaryson** [comment]
thanks for the review.

**Pull request merged**